js/app.js

```diff
@@ -532,6 +624,19 @@
 	*/
 
 	// implement function here
+  const combineLists = (items1, items2) => {
+
+    const list = items1.concat(items2);
+
+    list.forEach(item => {
+      if(typeof item !== 'object' && !item.hasOwnProperty('item') && !item.hasOwnProperty('price') &&
+          typeof item.price !== 'number' && typeof item.item !== 'string') {
+        throw new Error('Not a valid item');
+        return;
+      }
+    });
```

Comment:
Can we think about if there were _n_ items? Also, what if combining **item1** and **item2** breaks the structure3?